gips/data/modis/modis.py

```diff
@@ -1035,3 +1082,35 @@ def process(self, *args, **kwargs):
             del imgout  # to cover for GDAL's internal problems
             utils.verbose_out(' -> {}: processed in {}'.format(
                 os.path.basename(fname), datetime.datetime.now() - start), level=1)
+
+        # process some index products (not all, see above)
+        requested_tillage_pt = list(
+                set(products.requested.keys()) & set(_tillage_product_types))
+        if requested_tillage_pt:
+            model_pt = requested_tillage_pt[0] # all should be similar
+            asset, version, _, _, allsds = self.asset_check(model_pt)
+            if asset is None:
+                raise IOError('Found no assets for {}'.format(
+                    requested_tillage_pt))
+            if version < 6:
+                raise IOError('tillage products require product version 6,'
+                              ' but found {}'.format(version))
+            img = gippy.GeoImage(allsds[7:]) # don't need the QC bands
+
+            # GIPPY uses expected band names to find data
+            img.SetBandName('BLUE', 2) # array index 2 = band number 3
+            img.SetBandName('GREEN', 3)
+            img.SetBandName('SWIR1', 4)
+            img.SetBandName('SWIR2', 5)
```

Comment:
@bhbraswell please correct me if I'm wrong here, but I believe that this is what MODIS band ordering is:
```
# tuples are (COLOR, INDEX, CENTER_WAVELENGTH)
# where INDEX is out of the 7 band subset of MCD43A4 
[
    ('RED', 0, 655),
    ('NIR', 1,  858),
    ('BLUE', 2, 470),
    ('GREEN', 3, 555),
    ('CIRRUS', 4, 1240), 
    ('SWIR1', 5, 1640),
    ('SWIR2, 6, 2130),
]
```

Reply:
That does seem to match landsat 8's names for bands:

https://landsat.usgs.gov/what-are-band-designations-landsat-satellites